src/PowerShellLibrary/Microsoft.FactoryOrchestrator.Client.psd1

```diff
@@ -110,10 +106,10 @@ PrivateData = @{
         # ReleaseNotes = ''
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = '$VersionSuffix$'

```

Comment:
why 'prerelease'? or is that oinly if the versionsuffix exists? (can read docs too :D)

Reply:
Yeh this will allow me to use a build var to mark the build as prerelease. Nuget/NET call it a version suffix, PS calls it prerelease (https://docs.microsoft.com/en-us/powershell/scripting/gallery/concepts/module-prerelease-support?view=powershell-7.1). 

but syntax is Major.Minor.Patch-VersionSuffix for NET/Nuget (https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build, https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages)


Major.Minor.Patch == VersionPrefix

Reply:
So i figure I'll release our first nuget/PS as such to show it is not fully validated (9.0.0-RC1) then release a official ver (9.0.0) immediately after